UIKit_DZ/UIKit_DZ/FeedScreen/RecomendationsCell/RecomendationView.swift

```diff
@@ -0,0 +1,100 @@
+// RecomendationView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Вью используемая в качестве ячейки для раздела рекомендаций
+final class RecomendationView: UIView {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let subscribeText = "Подписаться"
+    }
+
+    // MARK: - Visual Components
+
+    private let imageView: UIImageView = {
```

Comment:
```suggestion
    private let imageView: UIImageView = {
```
imageView общее название